worlds/pokemon_rb/rock_tunnel.py

```diff
@@ -305,14 +340,6 @@ def check_addable_block(check_map, disallowed):
         current_map = map2f
         check_addable_block(map2f, disallowed2F)
 
-    address = rom_addresses["Map_Rock_Tunnel1F"]
-    for y in map1f:
-        for x in y:
-            data[address] = x
-            address += 1
-    address = rom_addresses["Map_Rock_TunnelB1F"]
-    for y in map2f:
-        for x in y:
-            data[address] = x
-            address += 1
-    return seed
\ No newline at end of file
+    patch.write_token(APTokenTypes.WRITE, rom_addresses["Map_Rock_Tunnel1F"], bytes(b for row in map1f for b in row))
+    patch.write_token(APTokenTypes.WRITE, rom_addresses["Map_Rock_TunnelB1F"], bytes(b for row in map2f for b in row))
```

Comment:
double check me but these can probably be faster with a comprehension than generator
```suggestion
    patch.write_token(APTokenTypes.WRITE, rom_addresses["Map_Rock_Tunnel1F"], bytes([b for row in map1f for b in row]))
    patch.write_token(APTokenTypes.WRITE, rom_addresses["Map_Rock_TunnelB1F"], bytes([b for row in map2f for b in row]))
```

Reply:
I hate how counterintuitive this is.